# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/slim/hotfix-string.patch
# Copyright (C) 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

Include <string> for std::string, no longer pulled in indirectly.

--- slim-1.4.1/app.h	2026-09-27 13:12:18.048312911 +0200
+++ slim-1.4.1-fixed/app.h	2026-09-27 13:12:18.052546949 +0200
@@ -13,6 +13,7 @@
 #ifndef _APP_H_
 #define _APP_H_
 
+#include <string>
 #include <X11/Xlib.h>
 
 #ifdef USE_PAM
